source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -8723,6 +8798,16 @@ export class AdaptiveCard extends ContainerWithActions {
                 if (this.speak) {
                     renderedCard.setAttribute("aria-label", this.speak);
                 }
+
+                // register for on mouse enter on the card
```

Comment:
Do we track these events so that `updateInputAdornersVisibility` will be called?

Reply:
I need to remove this comment. I had added for my previous implementation. Now It calls into updateInputsVisualState and that calls into Input.updateVisualState